controller/apod.controller.js

```diff
@@ -0,0 +1,60 @@
+const { Sequelize, DataTypes, Model } = require("sequelize");
+const database = require("../config/database.js");
+const sequelize = new Sequelize(database.development);
+const ApodModel = require("../db/models/apod.js");
+const Apod = ApodModel(sequelize, Sequelize);
+const axios = require("axios");
+const URL = process.env.URL_API + process.env.API_KEY;
+const URL_RANDOM = process.env.URL_API + process.env.API_KEY + "&count=1";
+
+async function getAPODData(url) {
+  try {
+    dataRecovered = await axios.get(url);
+    if (dataRecovered.data.length) {
```

Comment:
se nao for um array, a ausencia da propriedade `length` não vai quebrar esse `if` e entrar sem querer no `catch` ?

Reply:
Acho que retorna undefined. Fiz uns testes e não lança erro.